Sources/Views/AnimatableTableViewCell.swift

```diff
@@ -156,7 +159,9 @@ open var startPoint: GradientStartPoint = .top
   fileprivate func configureInspectableProperties() {
     configureAnimatableProperties()
     configureOpacity()
+    #if os(iOS)
```

Comment:
Can we make all changes in the protocols, then someone conforms to the protocols and deson't need to apply if os in their subclasses.